backend/app/usecase/instrumentation/instrumentation.go

```diff
@@ -91,6 +94,29 @@ func (i Instrumentation) FeatureToggleRetrievalFailed(err error) {
 	}()
 }
 
+// SearchSucceed tracks the successes when searching the short resources.
+func (i Instrumentation) SearchSucceed(user *entity.User, keywords string, t time.Time) {
+	go func() {
+		c := <-i.searchSucceedCh
+		i.metrics.Count("search-succeed", 1, 1, c)
+		userID := i.getUserID(user)
+		props := map[string]string{
+			"keywords": keywords,
+			"time":     t.String(),
```

Comment:
Time is automatically tracked by analytics API. I wonder whether we can track the list of `resources` being search also. Let's separate them with `,`.